scipy/optimize/__init__.py

```diff
@@ -200,7 +200,8 @@
 
 .. seealso::
 
-   `scipy.optimize.cython_optimize` -- Typed Cython versions of zeros functions
+   `scipy.optimize.cython_optimize` -- Typed Cython versions of root finding
+   functions
```

Comment:
Looks like this caused the refguide failure.

Reply:
Indeed, couldn't remember if you could split over multiple lines with this directive. Have fixed.